packages/@expo/config-plugins/src/Plugin.types.ts

```diff
@@ -112,9 +112,13 @@ export type Mod<Props = any> = ((
 export interface ModConfig {
   android?: {
     /**
-     * Dangerously make a modification before any other android mods have been run.
+     * Dangerously make a modification before any other platform mods have been run.
      */
     dangerous?: Mod<unknown>;
+    /**
+     * A finalized mod to run after all the other platform mods have been run.
+     */
+    finalized?: Mod<unknown>;
```

Comment:
There should be a helper config plugin like withDangerousMod that wraps this finalized mod